lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_helper.rb

```diff
@@ -6,60 +6,23 @@ module Fastlane
 
   module Helper
     module FirebaseAppDistributionHelper
-      def testers_flag(params)
-        file_flag_if_supplied("--testers-file", "testers", params)
-      end
-
-      def groups_flag(params)
-        file_flag_if_supplied("--groups-file", "groups", params)
-      end
-
-      def release_notes_flag(params)
-        file_flag_if_supplied("--release-notes-file", "release_notes", params)
-      end
-
-      def file_flag_if_supplied(flag, param_name, params)
-        file = params["#{param_name}_file".to_sym]
-        file ||= file_for_contents(param_name.to_sym, params)
-
-        if file
-          return "#{flag} #{file}"
+      def get_value_from_value_or_file(value, path)
+        if (value.nil? || value.empty?) && !path.nil?
+          begin
+            return File.open(path).read
+          rescue
+            UI.crash!("#{ErrorMessage::INVALID_PATH}: #{path}")
+          end
         end
+        value
       end
 
-      def flag_value_if_supplied(flag, param_name, params)
-        "#{flag} #{params[param_name]}" if params[param_name]
-      end
-
-      def flag_if_supplied(flag, param_name, params)
-        flag if params[param_name]
-      end
-
-      ##
-      # always return a file for a given content
-      def file_for_contents(parameter_name, params)
-        if @tempfiles.nil?
-          @tempfiles = []
+      def string_to_array(string)
+        if string.nil?
```

Comment:
Consider this Ruby-esque one-liner:
```suggestion
        return X if string.nil?
```